kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md

```diff
@@ -26,6 +26,12 @@ $ create-weblogic-credentials.sh
 
 ## Use the script to create a domain
 
+**Note**: The `create-domain.sh` script generates a new Docker image on each run with a new domain home and a different internal `domain secret` in it.  To prevent having disparate images with different domain secrets in the same domain, it is highly recommended that a new domain uses a `domainUID` that is different from any of the active domains, or that you delete the existing domain resource before you create a domain with the same `domainUID` using the following command (and wait until all the server pods are terminated):
```

Comment:
it is highly recommended -> we strongly recommend

(and wait until all the server pods are terminated): -> (_after_ all the server pods are terminated):